<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportItem</class>
 <widget class="QWidget" name="ReportItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>50</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>50</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget#ReportItem { background-color: transparent; }
    QLabel { font-family: &quot;Segoe UI&quot;; }
   </string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QLabel" name="lbl_name">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string>color: #475569; font-weight: 600; font-size: 13px;</string>
     </property>
     <property name="text">
      <string>Product Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progress_bar">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>12</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
        QProgressBar {
            background-color: #F1F5F9; /* Slate-100 */
            border-radius: 6px;
            border: none;
        }
        QProgressBar::chunk {
            background-color: #06B6D4; /* Cyan-500 */
            border-radius: 6px;
        }
      </string>
     </property>
     <property name="value">
      <number>80</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_count">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string>color: #0F172A; font-weight: 800; font-size: 13px;</string>
     </property>
     <property name="text">
      <string>80 sold</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
